examples/group_level/plot_continuous_covariate.py

```diff
@@ -0,0 +1,361 @@
+# we already have an intercept column in the design matrix,
+# thus we'll call LinearRegression with fit_intercept=False
+
+# set up and fit model
+linear_model = LinearRegression(fit_intercept=False)
+linear_model.fit(group_design, betas)
+
+# extract group-level beta coefficients
+group_coefs = get_coef(linear_model, 'coef_')
+
+# only keep relevant predictor
+group_betas = group_coefs[:, group_pred_col]
+
+# back projection to channels x time points
+group_betas = group_betas.reshape((n_channels, n_times))
+# create evoked object containing the back projected coefficients
+group_betas_evoked = EvokedArray(group_betas, epochs_info, tmin)
+
+###############################################################################
+# plot the modulating effect of age on the phase coherence predictor (i.e.,
+# how the effect of phase coherence varies as a function of subject age)
+# using whole electrode montage and whole scalp by taking the
+# same physical electrodes across subjects
+
+# index of B8 in array
+electrode = 'B8'
+pick = group_betas_evoked.ch_names.index(electrode)
+
+# create figure
+fig, ax = plt.subplots(figsize=(7, 4))
+ax = plot_compare_evokeds(group_betas_evoked,
+                          ylim=dict(eeg=[-3, 3]),
+                          picks=pick,
+                          show_sensors='upper right',
+                          axes=ax)
+ax[0].axes[0].fill_between(times,
+                           # transform values to microvolt
+                           upper_b[pick] * 1e6,
+                           lower_b[pick] * 1e6,
+                           alpha=0.2)
+plt.plot()
+
+###############################################################################
+# run analysis on optimized electrode (i.e., electrode showing best fit for
+# phase-coherence predictor).
+
+# reshape subjects' R-squared to channels * time-points space
+r_squared = r_squared.reshape((r_squared.shape[0], n_channels, n_times))
+
+# look for electrode with the highest R-squared variance in each subject
+optimized_electrode = []
+for i in range(r_squared.shape[0]):
+    subj_r2 = r_squared[i, :, :]
+
+    # use electrode R-squared variance
+    # electrode_variance = []
+    # for e in range(subj_r2.shape[0]):
+    #     electrode_variance.append(np.var(subj_r2[e, :]))
+
+    # optimized_electrode.append(np.argmax(electrode_variance))
+
+    # or use max R-squared ?
+    optimized_electrode.append(np.unravel_index(r_squared[i, :, :].argmax(),
+                                                r_squared[i, :, :].shape)[0])
+
+###############################################################################
+# extract beta coefficients for electrode showing best fit
+
+# reshape subjects' beats to channels * time-points space
+betas = betas.reshape((betas.shape[0], n_channels, n_times))
+
+optimized_electrode_betas = np.zeros((betas.shape[0], n_times))
+for i in range(betas.shape[0]):
+    optimized_electrode_betas[i, :] = betas[i, optimized_electrode[i], :]
```

Comment:
```suggestion
optimized_electrode_betas = np.array([subj_betas[elec, :] for subj_betas, elec in zip(betas, optimized_electrode)])
```

Wouldn't this work too? And you could have 247-273 in two lines.

Reply:
I like this one. Let's go for it